### DISCUSSION TITLE
Rabbit producer connection  throttle issue

### DISCUSSION THREAD

**Discussion by sandeepkr977** [rabbitmq/rabbitmq-server#11345] [Questions] [answered]
Hi,

We are using Rabbit MQ 3.11.22 & Erlang 25.3.2.6. 

We are getting producer connection throttle issue. The messages are producing at rate 540 message/sec and consumers are consuming at rate 850 message/sec. 1 message size is 200kb. RAM and Disk are completely free. 1 producer connection sending data to client network is 72 mb/sec.

Let us know any configuration which help to tune the Rabbit.

I have attached screenshot of rabbit MQ dashboard stats.

**Comment by lukebakken** [comment, marked as answer]
https://github.com/rabbitmq/rabbitmq-server/blob/main/COMMUNITY_SUPPORT.md#who-is-eligible-for-community-support
